### PR TITLE
[hi] Localize virtualization.md to Hindi

### LINKED ISSUE #1597: Virtualization For hindi

### PR DESCRIPTION
**Pull request by SaxenaAnushka102**
Signed-off-by: ANUSHKA SAXENA <anushkasaxena102@gmail.com>

### Describe your changes
Created a file `virualization.md` to localize the page: https://glossary.cncf.io/virtualization/ into Hindi 

### Related issue number 
Resolves #1597

### Checklist before opening this PR 
- [x] This PR does not contain plagiarism
- [x] I have signed off on all commits

### CODE REVIEW

File: content/hi/virtualization.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: आभासीकरण (Virtualization)
```
Comment (aj11anuj) [inline content/hi/virtualization.md:2]:
```suggestion
title: वर्चुअलाइजेशन (Virtualization)
```

File: content/hi/virtualization.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+category: संकल्पना
+tags:  ["Fundamental", "Infrastructure", "Methodology"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव कंप्यूटिंग के संदर्भ में आभासीकरण (वर्चुअलाइजेशन), एक भौतिक कंप्यूटर लेने की प्रक्रिया को संदर्भित करता है, जिसे कभी-कभी सर्वर भी कहा जाता है, और उसे कई पृथक ऑपरेटिंग सिस्टम चलाने की अनुमति देता है। उन पृथक ऑपरेटिंग सिस्टम और उनके समर्पित कंप्यूट संसाधन (सीपीयू, मेमोरी और नेटवर्क) को आभासी (वर्चुअल) मशीन या 'वीएम' (VM) कहा जाता है। जब हम आभासी मशीन के बारे में बात करते हैं, तो हम सॉफ्टवेयर-परिभाषित कंप्यूटर के बारे में बात कर रहे होते हैं। यह कुछ ऐसा है जो वास्तविक कंप्यूटर की तरह दिखता और कार्य तो करता है लेकिन अन्य आभासी मशीनों के साथ हार्डवेयर साझा कर रहा है। क्लाउड कंप्यूटिंग मुख्य रूप से आभासीकरण की तकनीक द्वारा संचालित है। उदाहरण के तौर पर, आप 'ए डब्ल्यू एस'(AWS) से "कंप्यूटर" किराये पर ले सकते हैं- यह कंप्यूटर वास्तव में एक आभासी मशीन है।
+
+## समस्या
+वर्चुअलाइजेशन कई समस्याओं का समाधान करता है, जिसमें सुरक्षा के लिए एक दूसरे से अलग होते हुए भी एक ही भौतिक मशीन पर और अधिक एप्लीकेशन को चलने की अनुमति देकर भौतिक हार्डवेयर उपयोग में सुधार भी शामिल है।
```
Comment (aj11anuj) [inline content/hi/virtualization.md:13]:
```suggestion
वर्चुअलाइजेशन कई समस्याओं का समाधान करता है, जिसमें सुरक्षा के लिए एक दूसरे से अलग होते हुए भी एक ही भौतिक मशीन पर और अधिक एप्लीकेशन को चलने की अनुमति देकर हार्डवेयर के उपयोग में सुधार भी कर सकते हैं।
```

File: content/hi/virtualization.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## समाधान
+
+आभासी मशीन पर चलने वाले विभिन्न एप्लीकेशन को इस बात की कोई जानकारी नहीं होती है कि वे एक भौतिक कंप्यूटर साझा कर रहे हैं। आभासीकरण डेटासेंटर के उपयोगकर्ताओं को डेटासेंटर में एक नया कंप्यूटर जोड़ने की भौतिक बाधाओं के बारे में चिंता किए बिना, मिनटों में एक नए "कंप्यूटर" (उर्फ़ वीएम) को तुरंत शुरू करने की अनुमति देता है। आभासी मशीन उपयोगकर्ताओं को एक नया आभासी कंप्यूटर प्राप्त करने के लिए लगने वाली समयावधि को घटाने में भी सक्षम बनाता है।
```
Comment (aj11anuj) [inline content/hi/virtualization.md:17]:
```suggestion
वर्चुअल मशीन पर चलने वाले विभिन्न एप्लीकेशन को इस बात की कोई जानकारी नहीं होती है कि वे एक भौतिक कंप्यूटर को साझा कर रहीं हैं। वर्चुअलाइजेशन डेटासेंटर के उपयोगकर्ताओं को डेटासेंटर में एक नया कंप्यूटर को जोड़ने की भौतिक बाधाओं के बारे में चिंता किए बिना, मिनटों में एक नए "कंप्यूटर" (वि एम) को तुरंत शुरू करने की अनुमति देता है। वर्चुअल मशीन उपयोगकर्ताओं को एक नया वर्चुअल कंप्यूटर प्राप्त करने के लिए लगने वाली समयावधि को घटाने में भी सक्षम बनाता है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Some suggestions from my side

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by kumarankit999 — APPROVED** [review]
lgtm

**Comment by bishal7679** [comment]
It seems this PR is ready to merge!
Just let me go through a final review once!

**Pull request merged**